### ISSUE TITLE
Installation error - databricks.sdk.errors.mapping.NotFound: Files in Repos must be enabled for AUTO ExportFormat

### ISSUE THREAD

**Issue by dipankarkush-db** [databrickslabs/ucx#576]
Here is the stack trace -

```
Enter a number between 0 and 2: 2
Comma-separated list of workspace group names to migrate. If not specified, we'll use all account-level groups with matching names to workspace-level groups. (default: <ALL>): 
Backup prefix (default: db-temp-): 
Log level (default: INFO): 
Number of threads (default: 8): INFO [__main__] Creating configuration file: /Users/XXXXX.XXXXX@domain.com/.ucx/config.yml
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/m3210483/source/ucx/src/databricks/labs/ucx/install.py", line 837, in <module>
    installer.run()
  File "/home/m3210483/source/ucx/src/databricks/labs/ucx/install.py", line 107, in run
    self._configure()
  File "/home/m3210483/source/ucx/src/databricks/labs/ucx/install.py", line 367, in _configure
    self._write_config()
  File "/home/m3210483/source/ucx/src/databricks/labs/ucx/install.py", line 383, in _write_config
    self._ws.workspace.upload(self.config_file, config_bytes, format=ImportFormat.AUTO)
  File "/tmp/tmp.RALkKV481G/lib/python3.10/site-packages/databricks/sdk/mixins/workspace.py", line 90, in upload
    raise e
  File "/tmp/tmp.RALkKV481G/lib/python3.10/site-packages/databricks/sdk/mixins/workspace.py", line 84, in upload
    return self._api.do('POST', '/api/2.0/workspace/import', files={'content': content}, data=data)
  File "/tmp/tmp.RALkKV481G/lib/python3.10/site-packages/databricks/sdk/core.py", line 1070, in do
    return retryable(self._perform)(method,
  File "/tmp/tmp.RALkKV481G/lib/python3.10/site-packages/databricks/sdk/retries.py", line 47, in wrapper
    raise err
  File "/tmp/tmp.RALkKV481G/lib/python3.10/site-packages/databricks/sdk/retries.py", line 29, in wrapper
    return func(*args, **kwargs)
  File "/tmp/tmp.RALkKV481G/lib/python3.10/site-packages/databricks/sdk/core.py", line 1162, in _perform
    raise self._make_nicer_error(response=response, **payload) from None
databricks.sdk.errors.mapping.NotFound: Files in Repos must be enabled for AUTO ExportFormat
```

**Comment by nfx** [comment]
@prajin-29 use this api to check if files-in-repos are enabled - https://databricks-sdk-py.readthedocs.io/en/latest/workspace/workspace_conf.html#WorkspaceConfAPI.get_status

**Comment by dipankarkush-db** [comment]
Here are the steps for resolving this issue -
```
1) Enable/Disable Repos
If you are already use Repos on the workspace, you can skip this, if you are not, please ensure that Repos is enabled by follow these steps: https://gist.github.com/simonfl-db/e4dcbe75fae769304d2f7e0cd0e734dd

2) Enable/Disable Files
You can use a notebook and the SDK for this one as well or execute the call on the command line as shown below (you'll have to set your DB_TOKEN and DB_URL env variables first, btw) :

curl -n -X PATCH -H "Authorization: Bearer $DB_TOKEN" -d '{ "enableWorkspaceFilesystem": "true" }' $DB_URL/api/2.0/workspace-conf | jq
```

**Comment by nfx** [comment]
we need to enable WSFS from UCX automatically

**Comment by nfx** [comment]
Fixed in databrickslabs/blueprint#42
